### PR TITLE
Improve gitlab integration tests logging on failure

### LINKED ISSUE #793: Improve integration test logging in gitlab
In failed pipelines, it is not possible to see the output of the failing tests.  The output goes to an error log and we should create a script or pipe some commands together to output any failing test outputs to the screen.

### PR DESCRIPTION
**Pull request by chapman39**
Fixes #793

### CODE REVIEW

**Comment by chapman39** [comment]
Ready for review. You can only see the results on `quartz-clang_10_0_0-src` if it fails. Once everything is ready to be merged I'll revert the `tests` submodule back to normal so everything will pass.

**Review by jamiebramwell — APPROVED** [review]
This is really nice! Thanks!

File: scripts/gitlab/build_and_test.sh
Comment (white238) [inline scripts/gitlab/build_and_test.sh]:
```suggestion
                    echo ; echo "======== START $f START ========"
```
Apparently echo doesn't like '\n'.  This is what you get for trusting random things I send you in teams that I didn't bother to test. 

Apparently you can also use `printf "\n foo"` but `echo -e "\n foo"` isn't on all *nix platforms.

**Review by white238 — APPROVED** [review]
Approved pending my last code suggestion and reverting the broken test (obviously).

Thanks @chapman39 !

**Pull request merged**
